pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/ProtobufSchemaTest.java

```diff
@@ -89,7 +89,7 @@ public void testGenericOf() {
         try {
             ProtobufSchema<org.apache.pulsar.client.schema.proto.Test.TestMessage> protobufSchema
                     = ProtobufSchema.ofGenericClass(org.apache.pulsar.client.schema.proto.Test.TestMessage.class,
-                    Collections.emptyMap());
+                    new HashMap<>());
```

Comment:
what is the reason changing `Collections.emptyMap` to `new HashMap`?

Reply:
same question here